### ISSUE TITLE
Can't Enchant Books with anything 

### ISSUE THREAD

**Issue by ainhall-assassin** [Epoxide-Software/Enchanting-Plus#141]
Well sort of. I can only enchant the books with Reading I from Bibliocraft. See pic:
https://i.imgur.com/xu01NmY.png

I perused the config file for both Bibliocraft and ePlus but found nothing relevant to enchantment restrictions for books. Is there a work around/fix that I could perform?

**Comment by Joccob** [comment]
Same problem here. Normally I wouldn't bother since I rarely use enchanting books anyway. But there are some mods in my modpack that use specific enchanting books in crafting recipe. It would be really nice if this can be fixed.

**Comment by grydian** [comment]
I would also like to second this.  Would be a nice addition.

**Comment by DeadlyArtist** [comment]
Yeah same here. Compared to 1.7.10 the config options in 1.12 are pretty much non-existent. Have seen comments on 2 issues that there are some unseen configurations you could theoretically add if I understood correctly, but seems like this mod has been forgotten. Still remember it when I first got to know it in Hexxit :(
